plugins/flytekit-influxdb/flytekitplugins/influxdb/task.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+from typing import Any, Dict, List
+
+from utils import Aggregation
+
+from flytekit.configuration import SerializationSettings
+from flytekit.core.base_task import PythonTask
+from flytekit.core.interface import Interface
+from flytekit.extend.backend.base_agent import SyncAgentExecutorMixin
+
+
+class InfluxDBTask(SyncAgentExecutorMixin, PythonTask):
+    _TASK_TYPE = "influxdb"
```

Comment:
Can we add annotations like chatgpt agent for better UX experience?

https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-openai/flytekitplugins/openai/chatgpt/task.py#L10-L12